src/plugins/unified_search/public/filter_bar/filter_bar.tsx

```diff
@@ -27,6 +27,7 @@ export interface Props {
    * Applies extra styles necessary when coupled with the query bar
    */
   afterQueryBar?: boolean;
+  readOnly?: boolean;
```

Comment:
changes for unified_search components will be removed after https://github.com/elastic/kibana/pull/140101 is merged, and present here only for demo/POC purposes